web_ui/prometheus.go

```diff
@@ -19,6 +19,7 @@ package web_ui
 import (
 	"context"
 	"fmt"
+	pelican_log "log"
```

Comment:
We already import `"github.com/sirupsen/logrus"`, which is the logger we use throughout Pelican, so let's use that instead. You should do:
```
log "github.com/sirupsen/logrus"

...

log.Warningf("Here's some message that says %s", <your specifier>)
```